src/sdp_core/contracts.py

```
@@ -17,9 +17,58 @@ class ColumnMetadata(BaseModel):
+            "id": self.distribution_id,
+            "format": self.distribution_format,
+            "endpoint": self.distribution_endpoint,
+        }
+
+    @property
+    def id(self) -> str:
+        """Return the legacy distribution identifier compatibility attribute."""
+
+        return self.distribution_id
+
+    @id.setter
```

Ruff (0.16.3)

[error] Python builtin is shadowed by method `id` from line 50

(A003)